Python.gitignore

```diff
@@ -43,6 +43,7 @@ htmlcov/
 nosetests.xml
 coverage.xml
 *,cover
+.hypothesis/
```

Comment:
I think this is wrong @kxepal . The trailing slash just means that it is a directory. A *leading* slash would mean to only ignore a file with this name in the root directory (or in any case, the directory which contains the `.gitignore`).